test/igniter/code/file_test.exs

```diff
@@ -0,0 +1,45 @@
+defmodule Igniter.Code.FileTest do
+  use ExUnit.Case
+
+  test "files will be created if they do not exist, in the conventional place, which can be configured" do
```

Comment:
The conventional place for non-elixir files can't be configured, so I'd just remove the `igniter_exs` code and make the name of the test just "files will be created if they do not exist"